manifests/b/BAR_Team/Beyond-All-Reason/1.1779.0/BAR_Team.Beyond-All-Reason.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: BAR_Team.Beyond-All-Reason
+PackageVersion: 1.1779.0
+PackageLocale: en-US
+Publisher: BAR Team

```

```suggestion
Publisher: gajop
```

This must match control panel